### PR TITLE
add tests for Kinetics400

### PR DESCRIPTION
**Pull request by pmeier**

### CODE REVIEW

File: test/test_datasets.py [hunk 1]
```diff
@@ -954,5 +955,24 @@ def _create_annotation_file(self, root, name, video_files):
...
+        letters = string.ascii_letters + string.digits + "-_"
+        for cls in classes:
+            datasets_utils.create_video_folder(
+                tmpdir, cls, lambda _: f"{''.join(random.choice(letters) for _ in range(11))}.avi", num_videos_per_class
+            )
+
+        return num_videos_per_class * len(classes)
+
+    def test_not_found_or_corrupted(self):
+        self.skipTest("Dataset currently does not handle the case of no found videos.")
```
Comment (pmeier) [inline test/test_datasets.py:974]: This can be removed after #2903 is fixed.

**Comment by vadimkantorov** [comment]
maybe related: https://github.com/pytorch/vision/issues/3273

**Comment by pmeier** [comment]
@vadimkantorov That would be #3453 :wink:

**Review by vfdev-5 — APPROVED** [review]
LGTM!

File: test/test_datasets.py [hunk 2]
```diff
@@ -1035,6 +1035,34 @@ def _create_lmdb(self, root, cls):
 
         return num_images
 
+    def test_not_found_or_corrupted(self):
+        # LSUN does not raise built-in exception, but a custom one. It is expressive enough to not 'cast' it to
+        # RuntimeError or FileNotFoundError that are normally checked by this test.
+        with self.assertRaises(datasets_utils.lazy_importer.lmdb.Error):
+            super().test_not_found_or_corrupted()
```
Comment (fmassa) [inline test/test_datasets.py:1042]: nit: this is an unrelated fix and would have been better fit for a separate PR

**Review by fmassa — APPROVED** [review]
